phpstan.neon

```diff
@@ -724,3 +723,58 @@ parameters:
 
         # intersection bug
         - '#Parameter \#1 \$phpDocTagValueNode of method Rector\\BetterPhpDocParser\\PhpDocInfo\\PhpDocInfo\:\:removeTagValueNodeFromNode\(\) expects PHPStan\\PhpDocParser\\Ast\\PhpDoc\\PhpDocTagValueNode, Rector\\PhpAttribute\\Contract\\PhpAttributableTagNodeInterface given#'
+
+
+        -
+            message: '#Method call argument on position 0 must use constant over value#'
+            paths:
+                - packages/node-name-resolver/src/NodeNameResolver/FuncCallNameResolver.php # 38
+                - packages/node-name-resolver/src/NodeNameResolver/FuncCallNameResolver.php # 38
+                - rules/legacy/src/Rector/Include_/RemoveIncludeRector.php # 56
+                - rules/legacy/src/Rector/Include_/RemoveIncludeRector.php # 56
+                - rules/php73/src/Rector/String_/SensitiveHereNowDocRector.php # 64
+                - rules/php73/src/Rector/String_/SensitiveHereNowDocRector.php # 64
+                - rules/php73/src/Rector/String_/SensitiveHereNowDocRector.php # 70
+                - rules/php73/src/Rector/String_/SensitiveHereNowDocRector.php # 70
+                - rules/symfony/src/Rector/ClassMethod/AutoWireWithClassNameSuffixForMethodWithRequiredAnnotationRector.php # 75
+                - rules/symfony/src/Rector/ClassMethod/AutoWireWithClassNameSuffixForMethodWithRequiredAnnotationRector.php # 75
+
+        -
+            message: '#Method with @required need to be named autowire\+class name#'
+            paths:
+                - packages/changes-reporting/src/Rector/AbstractRector/NotifyingRemovingNodeTrait.php # 27
+                - packages/changes-reporting/src/Rector/AbstractRector/NotifyingRemovingNodeTrait.php # 27
+                - packages/post-rector/src/Rector/AbstractRector/NodeCommandersTrait.php # 71
+                - packages/post-rector/src/Rector/AbstractRector/NodeCommandersTrait.php # 71
+                - packages/reporting/src/Rector/AbstractRector/NodeReportCollectorTrait.php # 24
+                - packages/reporting/src/Rector/AbstractRector/NodeReportCollectorTrait.php # 24
+                - rules/doctrine/src/AbstractRector/DoctrineTrait.php # 23
+                - rules/doctrine/src/AbstractRector/DoctrineTrait.php # 23
+                - rules/doctrine/src/AbstractRector/DoctrineTrait.php # 23
+                - src/Rector/AbstractRector/BetterStandardPrinterTrait.php # 37
+                - src/Rector/AbstractRector/BetterStandardPrinterTrait.php # 37
+                - src/Rector/AbstractRector/CallableNodeTraverserTrait.php # 24
+                - src/Rector/AbstractRector/CallableNodeTraverserTrait.php # 24
```

Comment:
I ignored new PHPStan errors. Should I create new issue for it? (I don't feel like a scout today :))

Reply:
We should have this covered in CI already

Reply:
I thought it is covered but CI didn't fix it for some reason....

Reply:
You probably haven't rebased on `master`

I removed it with no PHPStan report:
https://github.com/rectorphp/rector/pull/4354

Reply:
So you were right, it was a bug. My bad